cdci_data_analysis/flask_app/templates/body_article_product_gallery.py

```diff
@@ -0,0 +1,43 @@
+body_article = {
+    "_links": {
+        "type": {
+            "href": "http://cdciweb02.isdc.unige.ch/mmoda-pg/rest/type/node/"
```

Comment:
maybe we could put some generic url here to avoid showing with our internal stuff

Reply:
this will actually be extracted from the configuration, so it will go